### PR TITLE
Fix: updated docs for datalake-azure

### PR DESCRIPTION
**Pull request by NiharDoshi99**
### Describe your changes :

I worked on updating datalake-azure docs

#
### Type of change :

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

### CODE REVIEW

**Review by MilanBariya — DISMISSED** [review]

File: openmetadata-docs/content/connectors/database/datalake/airflow.md [hunk 1]
```diff
@@ -193,6 +199,49 @@ The `sourceConfig` is defined [here](https://github.com/open-metadata/OpenMetada
...
+#### Source Configuration - Service Connection using Azure
+
+The `sourceConfig` is defined [here](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/security/credentials/azureCredentials.json).
+
+- **Client ID** : Client ID of the data storage account
+- **Client Secret** : Client Secret of the account
+- **Tenant ID** : Tenant ID under which the data storage account falls
+- **Account Name** : Account Name of the data Storage
+
+**schemaFilterPattern** and **tableFilternPattern**: Note that the `schemaFilterPattern` and `tableFilterPattern` both support regex as `include` or `exclude`. E.g.,
```
Comment (ulixius9) [inline openmetadata-docs/content/connectors/database/datalake/airflow.md:243]: let's add some more details regarding filter patterns here that we are using schema filter pattern to filter out the buckets and table filter pattern is used to filter files

Comment (ulixius9) [reply openmetadata-docs/content/connectors/database/datalake/airflow.md:243]: This is missing for gcs and s3 as well so let's add it there as well

Comment (NiharDoshi99) [reply openmetadata-docs/content/connectors/database/datalake/airflow.md:243]: Ok . Let me do that.

**Review by TeddyCr — APPROVED** [review]
LGTM

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**